### ISSUE TITLE
CSS not inlined if same import is used in multiple html pages

### ISSUE THREAD

**Issue by harryzcy** [parcel-bundler/parcel#9388]
# 🐛 bug report

I have two pages that imports the same css file (the tailwind base.css file that contains `@tailwind` directives). I'm trying to inline CSS using ```<style type="text/css"> @import './base.css'; </style>```, but parcel generates a separate css file and uses `<link rel="stylesheet" href="/index.**.css">` instead.

## 🎛 Configuration (.babelrc, package.json, cli command)

```json
{
  "extends": "@parcel/config-default",
  "transformers": {}
}

```

## 🤔 Expected Behavior

CSS are inlines inside `style`.

## 😯 Current Behavior

CSS are loaded via `link` element.

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel   |     v2.10.3    |
| Node  | v20.9.0           |
| npm |  v10.2.1        |

**Comment by bradjpalmer** [comment]
I have just come across the same issue, in the same situation - multiple HTML pages, wanting all of them to bundle the CSS inline. Also using tailwind. 

I noticed that removing tailwind from the postcss config made it work as expected so I carried on digging. 

I came across this section in the Parcel docs (after a lot of google searching) - https://parceljs.org/languages/css/#postcss-import

My guess is that the 'tailwindcss' plugin uses the 'postcss-custom-properties' plugin, so we need to use the 'postcss-import' plugin.

In short, adding the 'postcss-import' plugin to my postcss config fixed it for me. 


My postcssrc.json file now looks like this:

```
{
  "plugins": {
    "postcss-import": true,
    "tailwindcss": {}
  }
}
```

Hope this helps you!

**Comment by harryzcy** [comment]
@Digetal Thanks. This solves the issue!
